types/stripe-js/elements/payment.d.ts

```diff
@@ -110,6 +110,24 @@ export type StripePaymentElement = StripeElementBase & {
   collapse(): StripePaymentElement;
 };
 
+export interface DefaultValuesOption {
+  billingDetails?:
+    {
+      name?: string;
+      email?: string;
+      phone?: string;
+      address?:
+        {
+          country?: string;
+          postalCode?: string;
```

Comment:
This should be `postal_code` instead of `postalCode` ([doc](https://stripe.com/docs/js/elements_object/create_payment_element#payment_element_create-options-defaultValues-billingDetails-address-postal_code))!

Reply:
@jackieosborn-stripe Thank you for the quick review and nice catch! It seems inconsistent, for example with the [`postalCode` in the `fields` option](https://github.com/stripe/stripe-js/blob/19cdc3ee89a788be9ea90fc5558fd2f59d952b77/types/stripe-js/elements/payment.d.ts#L126). I can update it if the 🐍  case is intentional!

Reply:
Updated it in [2ad16db](https://github.com/stripe/stripe-js/pull/332/commits/2ad16dbabac75af29658ad94c4cf87c8272d1875).

Reply:
Thanks for bringing it up - we're aware of the inconsistencies 🙂